integration/check_test.go

```diff
@@ -34,25 +34,21 @@ type SkopeoSuite struct {
 }
 
 func (s *SkopeoSuite) SetUpSuite(c *check.C) {
-
-}
-
-func (s *SkopeoSuite) TearDownSuite(c *check.C) {
-
-}
-
-func (s *SkopeoSuite) SetUpTest(c *check.C) {
 	_, err := exec.LookPath(skopeoBinary)
 	c.Assert(err, check.IsNil)
 
-	s.regV1 = setupRegistryV1At(c, privateRegistryURL0, false) // TODO:(runcom)
+	if false { // not used
+		s.regV1 = setupRegistryV1At(c, privateRegistryURL0, false) // TODO:(runcom)
+	}
 	s.regV2 = setupRegistryV2At(c, privateRegistryURL1, false, false)
-	s.regV2Shema1 = setupRegistryV2At(c, privateRegistryURL2, false, true)
-	s.regV1WithAuth = setupRegistryV1At(c, privateRegistryURL3, true) // not used
+	if false { // not used
+		s.regV2Shema1 = setupRegistryV2At(c, privateRegistryURL2, false, true)
```

Comment:
Yes, `CopySuite.TestCopySchemaConversion` tests both OpenShift and non-OpenShift registries, and relies on registries set up in `CopySuite.SetUpSuite`, not on this one.